src/worker/tasks/sendTransactionWorker.ts

```diff
@@ -95,9 +99,48 @@ const handler: Processor<any, void, string> = async (job: Job<string>) => {
 
   if (transaction.status === "queued") {
     if (transaction.isUserOp) {
-      resultTransaction = await _sendUserOp(job, transaction);
+      // this can either be a regular backend wallet userop or a smart backend wallet userop
+      const accountAddress = transaction.accountAddress;
+      assert(accountAddress, "Invalid userOp parameters: accountAddress");
+
+      let adminAccount: Account | undefined;
+
+      try {
+        adminAccount = await getSmartBackendWalletAdminAccount({
+          accountAddress,
+          chainId: transaction.chainId,
+        });
+      } catch {
+        // do nothing, this might still be a regular backend wallet userop
+      }
+
+      try {
+        adminAccount = await getAccount({
+          chainId: transaction.chainId,
+          from: transaction.from,
+        });
+      } catch {
+        job.log(
+          `Failed to get admin account for userOp: ${stringify(transaction)}`,
+        );
+      }
+
+      if (!adminAccount) {
+        resultTransaction = {
+          ...transaction,
+          status: "errored",
+          errorMessage: "Failed to get admin account",
+        };
+      } else {
+        resultTransaction = await _sendUserOp(job, transaction, adminAccount);
+      }
     } else {
-      resultTransaction = await _sendTransaction(job, transaction);
+      const account = await getAccount({
+        chainId: transaction.chainId,
+        from: transaction.from,
+      });
+
+      resultTransaction = await _sendTransaction(job, transaction, account);
```

Comment:
It didn't seem like we needed to change this: getAccount can still be called inside _sendTransaction. If there's not a good reason to, let's revert this change.